pkg/roachprod/install/install.go

```diff
@@ -93,6 +97,19 @@ echo "deb [signed-by=/etc/apt/keyrings/fluent-bit.gpg] https://packages.fluentbi
 sudo apt-get update;
 sudo apt-get install -y fluent-bit;
 `,
+
+	"side-eye": `
```

Comment:
could you write a docstring here that lays out how to install side eye? this could also go in the commit message. i.e. i think it is something like, but it's not totally clear from the code:
```
roachprod install $CLUSTER side-eye --secret $API_KEY
```

Reply:
It is just `roachprod install TARGET side-eye` same as all the others, eg `roachprod install TARGET haproxy`. Given it is consistent with all the other install commands it seems weird to give it special documentation?

Reply:
Ah, so the user needs to set the `SIDE_EYE_API_TOKEN` env var. I think that's worth documenting.

Reply:
i'm just thinking through ways to ease the onboarding process for any CRL engineer. Ideally they need not ask questions about how to install the tool.

Reply:
> Ah, so the user needs to set the `SIDE_EYE_API_TOKEN` env var. I think that's worth documenting.

No, the key is fetched from gcloud (which you already need to have configured to use roachprod); the user with keys on keyboard never touches the key.

Reply:
> i'm just thinking through ways to ease the onboarding process for any CRL engineer.

Again, it is identical to all the other install commands, which seems like it should be as easy as we can do

Reply:
> i'm just thinking through ways to ease the onboarding process for any CRL engineer.

Again, it is identical to all the other install commands, which seems like it should be as easy as we can do

Reply:
I see, sorry, was not reading the code as closely as I could have.